Sources/ArcGISToolkit/Components/AR/FlyoverSceneView.swift

```diff
@@ -44,35 +45,22 @@ public struct FlyoverSceneView: View {
         let cameraController = TransformationMatrixCameraController(originCamera: initialCamera)
         cameraController.translationFactor = translationFactor
         _cameraController = .init(initialValue: cameraController)
-        
-        configuration = ARWorldTrackingConfiguration()
-        configuration.worldAlignment = .gravityAndHeading
     }
     
     public var body: some View {
-        ZStack {
-            SceneViewReader { sceneViewProxy in
-                sceneViewBuilder(sceneViewProxy)
-                    .cameraController(cameraController)
-                    .viewDrawingMode(.manual)
-                ARSwiftUIView(proxy: arViewProxy)
-                    .onRender { _, _, _ in
-                        updateLastGoodDeviceOrientation()
-                        sceneViewProxy.draw(
-                            for: arViewProxy,
-                            cameraController: cameraController,
-                            orientation: lastGoodDeviceOrientation
-                        )
-                    }
-                    .videoFeedHidden()
-                    .disabled(true)
-                    .onAppear {
-                        arViewProxy.session?.run(configuration)
-                    }
-                    .onDisappear {
-                        arViewProxy.session?.pause()
-                    }
-            }
+        SceneViewReader { sceneViewProxy in
+            sceneViewBuilder(sceneViewProxy)
+                .cameraController(cameraController)
+                .onAppear { session.start() }
+                .onDisappear { session.pause() }
+                .onChange(of: session.currentFrame) { frame in
+                    guard let frame else { return }
+                    sceneViewProxy.updateCamera(
```

Comment:
```suggestion
                    guard let frame else { return }
                    updateLastGoodDeviceOrientation()
                    sceneViewProxy.updateCamera(
```
This line was removed so the device orientation was not updating.